dsl/com/electriccloud/commander/dsl/util/BaseProject.groovy

```diff
@@ -132,9 +132,10 @@ abstract class BaseProject extends DslDelegatingScript {
       pipesDir.eachDir {dlist << it}
       dlist.sort({it.name})
       for (fdir in dlist) {
+        def pipeName=fdir.name
         File dslFile = getObjectDSLFile(fdir, "pipeline")
         if (dslFile?.exists()) {
-          println "Processing pipeline DSL file ${dslFile.absolutePath}"
+          println "Processing pipeline file projects/$projectName/pipelines/$pipename/${dslFile.name}"
```

Comment:
typo $pipename should be $pipeName

Reply:
Fixed. Need more tests.
I ran an import successfully
Will add specs tests